src/api/client.ts

```diff
@@ -82,6 +83,16 @@ export function getNostoClient(): PromiseLike<NostoClient> {
     })
 }
 
+export function logAndCaptureError(error: unknown, level: 'debug' | 'info' | 'warn' | 'error') {
```

Comment:
how about

```
log(msg: string, level: level: 'debug' | 'info' | 'warn' | 'error')
log(msg: string, error: unknown, level: 'debug' | 'info' | 'warn' | 'error')
```

which uses api.captureError & console internally?

Reply:
later, when the client script side logging endpoint is available you can replace console usage with the client script logging endpoint